ipatests/test_integration/test_trust.py

```diff
@@ -106,6 +108,45 @@ def check_range_properties(self, realm, expected_type, expected_size):
         expected_text = 'iparangetype: %s\n' % expected_type
         assert expected_text in result.stdout_text
 
+    def mod_idrange_auto_private_group(
+        self, auto_priv_grp_val='false'
+    ):
+        """
+        Set the auto-private-group option of the default trusted
+        AD domain range and restart sssd
+        """
+        rangename = self.existing_rangename
+        error_msg = "ipa: ERROR: no modifications to be performed"
+        cmd = ["ipa", "idrange-mod", rangename,
+               "--auto-private-groups", auto_priv_grp_val]
+        result = self.master.run_command(cmd, raiseonerr=False)
+        try:
+            self.master.run_command(["systemctl", "restart", "sssd"])
+        except subprocess.CalledProcessError as e:
+            # ignore error if the auto-private-group is already set
+            # to the expected value
+            if error_msg not in e.stderr:
+                raise
+
+    def get_user_id(host, username):
```

Comment:
Either the method is defined in this class as a static method, and it needs to be 
```
@staticmethod
def get_user_id(host, username):
```
or it is a function defined outside of any class, and it can be `def get_user_id(host, username):`.